src/test/java/uk/nhs/digital/nhsconnect/nhais/repository/SequenceRepositoryTest.java

```diff
@@ -38,7 +40,7 @@ public void When_GetNextKey_Expect_CorrectValue() {
     public void When_GetMaxNextKey_Expect_ValueReset() {
         when(mongoOperations.findAndModify(any(Query.class), any(Update.class), any(FindAndModifyOptions.class),
                 eq(OutboundSequenceId.class)))
-                .thenReturn(new OutboundSequenceId(MAX_KEY, 10000000L))
+                .thenReturn(new OutboundSequenceId(MAX_KEY, 100000000L))
                 .thenReturn(new OutboundSequenceId(MAX_KEY, 1L));
```

Comment:
I don't think this test is totally correct. The second thenReturn should be `100000001L`. We don't "wrap" the sequence number in the database only in the code.

Reply:
`100.000.000` is the first invalid number. `99.999.999` is the last valid one
Modulo 100M sounds ok here